resources/resinos.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Documentation    This resource provides access to host OS.
+Library   Process
+Library   OperatingSystem
+Library   String
+Library   Collections
+
+*** Variables ***
+
+*** Keywords ***
+Set up loop device
+    ${result} =  Run Process    losetup -f -P /autohat/resin.img    shell=yes
+    Process ${result}
+
+Find the last loop device that was assigned
+    ${result} =  Run Process    lsblk | grep 7: | tail -1 | awk -F: '{ print $2 }' | cut -d ' ' -f 1    shell=yes
+    Process ${result}
+    Should Not Be Empty     ${result.stdout}   msg="Couldn't find the loop device"
+    Set Suite Variable    ${LOOPDEVICE}    ${result.stdout}
+
+Mount file system using loop device
+    ${result} =  Run Process    mount /dev2/loop${LOOPDEVICE}p2 /mnt      shell=yes
+    Process ${result}
+    Directory Should Not Be Empty   /mnt    msg="Couldn't mount loop device to /mnt"
+
+Compare every md5sum of the host OS to resin-root.fingerprint file
+    File Should Exist   /mnt/resin-root.fingerprint     msg="Couldn't find resin-root.fingerprint file"
+    ${content} =  Get File  /mnt/resin-root.fingerprint
+    @{lines} =  Split To Lines  ${content}
+    : FOR   ${line}     IN  @{lines}
+    \   @{words} =  Split String    ${line}     ${SPACE}
+    \   ${First} =  Get From List   ${words}    0
+    \   ${Second} =  Get From List  ${words}    2
+    \   ${md5sum} =  Run Process    md5sum /mnt/${Second} | awk '{print $1}'     shell=yes
+    \   Should Contain     ${First}   ${md5sum.stdout}    msg=/mnt${Second} has MD5=${md5sum.stdout} when it should be ${First})
+
+Unmount file system using the loop device
```

Comment:
I think this keyword should take the path of the mount point to be unmounted as an input.